extensions/amp-access-laterpay/0.1/amp-access-laterpay.js

```diff
@@ -20,8 +20,10 @@ import {Services} from '../../../src/services';
 AMP.extension('amp-access-laterpay', '0.1', function(AMP) {
   AMP.registerServiceForDoc(
       'laterpay',
+      /** @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc */
```

Comment:
Is this really needed? Isn't `registerServiceForDoc` defined with this type already?

Reply:
I think since we're calling off of the `AMP` global, it uses the extern definition which has no type data. 

I can type the extern but I'd also have to extern `AmpDoc` type. Probably worth doing for all of the functions here: https://github.com/ampproject/amphtml/blob/b03c6fdf778fb07bf4ff87f9617f9bccd9fa5462/build-system/amp.extern.js#L196-L218